src/libs/PusherUtils.js

```diff
@@ -40,7 +40,7 @@ function subscribeToPrivateUserChannelEvent(eventName, accountID, onEvent, isChu
     function onSubscriptionFailed(error) {
         Log.hmmm('Failed to subscribe to Pusher channel', false, {error, pusherChannelName, eventName});
     }
-    Pusher.subscribe(pusherChannelName, eventName, onEventPush, isChunked, onPusherResubscribeToPrivateUserChannel)
+    Pusher.subscribe(pusherChannelName, eventName, onEventPush, onPusherResubscribeToPrivateUserChannel)
```

Comment:
Need to remove `isChunked` from this method signature

Reply:
Also we don't need to deal with it now but these nested functions are a bit odd. Not something we normally go for in our JS style

Reply:
Good call, removed from that method & where that param was sent in calling methods